docker-compose.yml

```diff
@@ -16,6 +16,7 @@ services:
         links:
             - mongo
             - redis
+        stop_grace_period: 60s    
```

Comment:
WDYT about giving it a few more seconds so that we can see the final messages of the shutdown process before docker kills the container force-fully?

Reply:
We should see the final message, as this is just the timeout. I think we'll have to tweak this, the 60s choice is a bit arbitrary (I've only tested in mac)